src/main/java/emissary/admin/Startup.java

```diff
@@ -421,20 +422,27 @@ protected void stopAndWaitForPlaceCreation() {
             numPlacesFound = this.places.size();
 
             if (numPlacesFound >= numPlacesExpected) {
+                boolean failedPlaceStartups = false;
 
                 if (!this.failedPlaces.isEmpty()) {
-                    String failedPlaceList = "The following places have failed to start: "
-                            + String.join(";", this.failedPlaces);
-                    logger.warn(failedPlaceList);
-                    if (this.node.isStrictStartupMode()) {
-                        logger.error(
-                                "Server failed to start due to Strict mode being enabled.  To disable strict mode, " +
-                                        "run server start command without the --strict flag");
-                        logger.error("Server shutting down");
-                        System.exit(1);
-                    }
+                    failedPlaceStartups = true;
+                    String failedPlaceList = String.join("; ", this.failedPlaces);
+                    logger.warn("The following places have failed to start: {}", failedPlaceList);
+                }
+                if (!CoordinationPlace.getFailedCoordinationPlaces().isEmpty()) {
+                    failedPlaceStartups = true;
+                    String failedCoordPlaceList = String.join("; ", CoordinationPlace.getFailedCoordinationPlaces());
+                    logger.warn("The following coordination places have failed to start: {}", failedCoordPlaceList);
```

Comment:
This logs the actual place that failed to start, not the coordination place. Do we want to:
1. Log the coordination place
2. Log the place that failed to start
3. Log both??

Reply:
Something I didn't even think about in the moment (😅 _sigh_). Personally, I think both is probably the best option, so we can point to the coordination place and provide the service coordination place that is explicitly causing the issue.
If you agree I can go in and make that change.